content/library/api/connections/connection.md

```diff
@@ -0,0 +1,16 @@
+---
+title: st.connection
+slug: /library/api-reference/connections/st.connection
+---
+
+<Tip>
+
+This page only contains on the `st.connection` API. For a deeper dive into creating and managing data connections within Streamlit apps, read [Connecting to data](/library/advanced-features/connecting-to-data).
```

Comment:
Typo: should be "only contains the `st.connection` API` (extra "on")

Reply:
(looks like other pages with this tip have the same typo)

Reply:
Ah! Copy pasta. Thanks.